analyzers/maven/maven.go

```diff
@@ -1,6 +1,7 @@
 // Package maven implements Maven analysis.
 //
-// A `BuildTarget` for Maven is the Maven project name.
+// A `BuildTarget` for Maven is a reactor build target
+// specified by [groupId]:artifactId or by its relative path
```

Comment:
When is relative path used as a `BuildTarget`?

Reply:
This is taken from the `mvn dependency:tree` docs

Reply:
BuildTarget is not going to be very helpful anymore, partly because of how it plays these two different roles. I'm working on a fix that relies on the `Module.Dir` property instead.